src/react-webcam.tsx

```diff
@@ -34,7 +35,8 @@ export default class Webcam extends React.Component<WebcamProps & React.HTMLAttr
     onUserMedia: () => {},
     onUserMediaError: () => {},
     screenshotFormat: "image/webp",
-    screenshotQuality: 0.92
+    screenshotQuality: 0.92,
+    mirrored: false
```

Comment:
Keep these in alphabetical order.

Reply:
I didn't realise that it was in alphabetical order 😄